src/main/java/lotto/domain/PrizeCalculator.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain;
+
+
+import java.util.Map;
+
+public class PrizeCalculator {
+
+    private final static double TICKET_PRICE = 1000.0d;
+
+    public static double calculate(Map<Prize, Integer> result) {
+         Long total = result.entrySet().stream()
+             .map((entry) -> entry.getKey().getReward() * entry.getValue())
+             .reduce(0L, Long::sum);
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같네요. 😃
```suggestion
         long total = result.entrySet().stream()
             .mapToLong((entry) -> entry.getKey().getReward() * entry.getValue())
             .sum();
```
추가적으로 ` entry.getKey().getReward() * entry.getValue()`  이 로직은 `Prize` 객체에게 [메시지를 보내는 방식](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)으로 변경해보면 어떨까요? 🙂

Reply:
반영했습니다 :)